paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/reporting/PackagePageRenderer.kt

```diff
@@ -0,0 +1,71 @@
+package app.cash.paparazzi.gradle.reporting
+
+import java.io.IOException
+
+/**
+ * Custom PackagePageRenderer based on Gradle's PackagePageRenderer
+ */
+internal class PackagePageRenderer : PageRenderer<PackageTestResults>() {
+
+  @Throws(IOException::class)
+  override fun renderBreadcrumbs(htmlWriter: SimpleHtmlWriter) {
+    htmlWriter.startElement("div").attribute("class", "breadcrumbs")
+    htmlWriter.startElement("a").attribute("href", "index.html").characters("all").endElement()
+    htmlWriter.characters(String.format(" > %s", results.name))
+    htmlWriter.endElement()
+  }
+
+  @Throws(IOException::class)
+  private fun renderClasses(htmlWriter: SimpleHtmlWriter) {
+    htmlWriter.startElement("table")
```

Comment:
Much of this writer logic is similar in this file and the `OverviewPageRenderer`. Would it make sense to extract the logic to extension functions on the html writer so it can be reused in multiple places?

This would also help spot differences and make sure they're intended. One case I can see is in the `OverviewPageRenderer`, after the `thead` element has been ended, it adds a `tbody` element but this renderer does not. It is difficult to determine if this is an intended difference or not.

Reply:
ya again i might keep this as is to not differentiate to much from the gradle source